modules/aardvarkBidAdapter.js

```diff
@@ -0,0 +1,154 @@
+import * as utils from 'src/utils';
+import {registerBidder} from 'src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'aardvark';
+const DEFAULT_ENDPOINT = 'bidder.rtk.io';
+const SYNC_ENDPOINT = 'sync.rtk.io';
+const AARDVARK_TTL = 300;
+const AARDVARK_CURRENCY = 'USD';
+
+var hasSynced = false;
+
+export function resetUserSync() {
+  hasSynced = false;
+}
+
+export const spec = {
+  code: BIDDER_CODE,
+
+  isBidRequestValid: function(bid) {
+    return ((typeof bid.params.ai === 'string') && !!bid.params.ai.length &&
+        (typeof bid.params.sc === 'string') && !!bid.params.sc.length);
+  },
+
+  buildRequests: function(validBidRequests, bidderRequest) {
+    var auctionCodes = [];
+    var requests = [];
+    var requestsMap = {};
+    var referer = utils.getTopWindowUrl();
+    var pageCategories = [];
+
+    if (window.rtkcategories && Array.isArray(window.rtkcategories)) {
```

Comment:
Can you clarify the use-case here for reading this window object?

Reply:
This is a requirement from several tier 1 publishers in order to allow RTK buyers to target specific inventory. It is currently implemented across Hearst and others